bot/package.json

```diff
@@ -1,34 +1,37 @@
 {
   "name": "airhornbot",
   "version": "1.0.0",
+  "description": "",
   "main": "index.js",
-  "author": "Discord",
+  "author": "Sky W",
   "license": "MIT",
-  "private": true,
+  "type": "module",
   "scripts": {
-    "register-commands": "node ./dist/register",
-    "bot": "node ./dist/bot",
-    "web": "node ./dist/web",
-    "build": "tsc --project ./tsconfig.json",
-    "lint": "eslint . --ext .ts,.tsx"
+    "build": "tsc --project tsconfig.json",
+    "bot": "node --enable-source-maps --experimental-specifier-resolution=node ./dist/bot.js",
```

Comment:
Huh - why do we need the `--experimental-specifier-resolution` flag here?  Maybe I'll learn below!

Reply:
This is required since I am using [esm](https://nodejs.org/api/esm.html) instead of commonjs. I could get away with using commonjs, but esm is becoming more and more common nowadays so I might as well switch now so there are no issues when using esm modules. (`node-fetch` is one of the major players, we don't use that package here, but I'm starting to default to esm.)